src/components/App.jsx

```diff
@@ -18,6 +30,24 @@ import TripAccount from 'src/components/TripAccount'
 import TripView from 'src/components/Trip'
 import Settings from 'src/components/Settings'
 
+const Content = ({ children }) => {
+  const { isMobile } = useBreakpoints()
+  const { pathname } = useLocation()
+
+  const isFullPage = pathname.startsWith('/trip/')
```

Comment:
Is it really the right implementation? Or do we have implemented it that way because our Dialog is not designed for this use case? 

I really think that if we go fullscreen we should do it with a Dialog / Modal. Not doing it manually.

Reply:
Je n'ai pas du tout cherché à utiliser Dialog donc je ne sais pas si ça pourrait causer des soucis. Je pense que non a priori.
Sur Desktop l'idée est d'avoir un style proche de celui de google maps (ce qui ne conviendrait pas je pense à une dialog), c'est pourquoi je n'ai pas pensé à Dialog (même si on pourrait utiliser le dialog ou non selon le breakpoint), et aussi parce que je trouve bizarre d'avoir un dialog dans ce cas... mais bon ok pour utiliser ce comp sur les fullscreen sur mobile.

La refacto ne sera pas pour cette PR par contre